capi/VideoDecoderCapi.cpp

```diff
@@ -18,8 +18,8 @@
 #include "config.h"
 #endif
 #include "VideoDecoderCapi.h"
-#include "interface/VideoDecoderInterface.h"
-#include "interface/VideoDecoderHost.h"
+#include "VideoDecoderInterface.h"
+#include "VideoDecoderHost.h"
```

Comment:
For the public headers, we have to remove `interface` from the include path.  For local library compilation units (.cpp) it doesn't matter so much.  However, it is better to be consistent across the board and less confusing.  So I'd prefer we "include" using the same syntax as the headers.  So this change here looks ok to me.  However, I agree with @chivakker that we should fix this across the entire project, too, and not just the capi files.